src/Shell/Task/ModelTask.php

```diff
@@ -953,15 +953,13 @@ protected function _getAllTables()
             $this->err(
                 'Connections need to implement schemaCollection() to be used with bake.'
             );
-
-            return $this->_stop();
+            $this->_stop();
```

Comment:
We were returning here to make the behavior consistent when `_stop()` is mocked.